Makefile

```diff
@@ -33,6 +33,10 @@ build-tests:
 	# connect to a server.
 	go test -short $(BUILD_TAGS) -run ^$$ ./...
 
+.PHONY: build-compile-check
+build-compile-check:
+	cd internal/test/compilecheck && go build $(BUILD_TAGS) && go mod tidy
```

Comment:
Here is a patch build showing that this would fail with a broken import: https://spruce.mongodb.com/task/mongo_go_driver_build_check_go1.13_build_patch_34cb4095341a85cb6f87ea04de55a53c7a532ba4_64529fd3306615eebfa87bdf_23_05_03_17_56_54/logs?execution=0